src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import java.util.ArrayList;
+
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.UI;
+import duke.task.Task;
+
+public class FindCommand implements Command {
+    private final String keyword;
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public String execute(TaskList taskList, UI ui, Storage storage) {
+        ArrayList<Task> list = taskList.getTaskList();
+        ArrayList<Task> foundList = new ArrayList<>();
```

Comment:
Perhaps a clearer naming could be used here? I feel that `foundList` sounds like a vague name that can be used for both variables and methods.